src/scss/components/_deprecated-select.scss

```diff
@@ -0,0 +1,127 @@
+@use 'sass:math';
+
+@import '~microscope-sass/lib/color';
+@import '~microscope-sass/lib/grid';
+@import '~microscope-sass/lib/transform';
+@import '~microscope-sass/lib/typography';
+
+@import '../mixins/prefix';
+
+$select-background-color: var(--of-select-background-color, $color-white);
+
+// Styling for the html5 widget for the select component
+.#{prefix(select)} {
```

Comment:
I think we were not using html5 widget anymore? :thinking: Not sure if we re-introduced it later: https://github.com/open-formulieren/open-forms/issues/1090

Reply:
checked it and it did indeed seem related to the non-choices.js version, so I deleted the styles and we'll see if something breaks